libs/cart/state/src/selectors/cart/cart.selector.ts

```diff
@@ -384,18 +377,18 @@ const createCartSelectors = <
   );
   const selectCartFeatureMutating = createSelector(
 		selectCartLoadingObject,
-		loadingObject => [
+		selectCartItemMutating,
+		(loadingObject, cartItemMutating) => [
       selectCartMutating,
       selectBillingAddressMutating,
       selectShippingAddressMutating,
       selectShippingInformationMutating,
       selectPaymentMutating,
       selectCouponMutating,
-			selectItemMutating,
-			selectItemAdding,
+			selectItemAdding
     ].map(selector =>
       selector.projector(loadingObject)
-    ).reduce((acc, mutating) => acc || mutating, false)
+    ).reduce((acc, mutating) => acc || mutating || cartItemMutating, false)
```

Comment:
`cartItemMutating` shouldn't be inside the `reduce` callback. It can either be the initial value for `reduce` or outside the `reduce` call entirely.

Reply:
Moved to outside the `reduce`